packages/azos-ui/vcl/cforest/settings-dialog.js

```diff
@@ -71,66 +73,32 @@ class ForestSettingsDialog extends ModalDialog {
   /**
    * Handles the click event for the Close button.
    * Resets the modal result to null and closes the dialog.
-   */
+   * Clears the forest and tree selections, and resets the asOf date input.
+  */
   #btnCancelClick(){
+    // Overly aggressive cleanup of previous settings on close.
+    this.selForest.innerHTML = "";
+    this.selForest.value = undefined;
+    this.selTree.innerHTML = "";
+    this.selTree.value = undefined;
+    this.tbAsOf.value = undefined;
+
     this.modalResult = null;
     this.close();
   }
 
-  /**
-   * Converts a forest option object to an HTML option element.
-   * @param {Object} opt - The forest option object containing id and title.
-   * @param {boolean} [selected=false] - Whether the option should be selected.
-   * @returns {HTMLTemplateElement} - The HTML option element.
-   */
-  #optToOption(opt, selected = false) {
-    return html`<option value="${opt.id}" .selected="${selected}" title="${opt.id}">${opt.id}</option>`;
-  }
-
-  renderBodyContent(){
-
-    const currentForestId = this.#tmpForestSelection || this.settings.activeForest;
-    const currentTreeId = this.#tmpTreeSelection || this.settings.activeTree;
-    const asOfValue = this.#tmpAsOfUtc || this.settings.activeAsOfUtc || undefined;
-
-    const forestOptions = this.settings.forests.map(forest => this.#optToOption(forest, currentForestId === forest.id));
-    const treeOptions = this.settings.forests.find(f => f.id === currentForestId)?.trees?.map(tree => this.#optToOption({ id: tree, title: tree }, currentTreeId === tree));
-
+  renderBodyContent() {
     return html`
       <div class="strip-h">
-        <az-select
-          id="selForest"
-          scope="this"
-          title="Forest"
-          rank="Normal"
-          @change="${this.#onForestChange}"
-          .value="${currentForestId}"
-          isRequired>${forestOptions}</az-select></div>
-
+        <az-select id="selForest" scope="this" title="Forest" rank="Normal" @change="${this.#onForestChange}" isRequired></az-select>
+      </div>
       <div class="strip-h">
-        <az-select
-          id="selTree"
-          scope="this"
-          title="Tree"
-          rank="Normal"
-          .value="${currentTreeId}"
-          isRequired>${treeOptions}</az-select></div>
-
+        <az-select id="selTree" scope="this" title="Tree" rank="Normal" isRequired></az-select>
+      </div>
       <div class="strip-h">
-        <az-text
-          id="tbAsOf"
-          scope="this"
-          title="As of Date"
-          placeholder="01/21/2022 1:00 pm"
-          dataType="date"
-          datakind="datetime"
-          timeZone="UTC"
-          .value="${asOfValue}"
-        ></az-text>
+        <az-text  id="tbAsOf" scope="this"  title="As of Date"  placeholder="01/21/2022 1:00 pm" dataType="date"  datakind="datetime"  timeZone="UTC" .value=${this.modalArgs?.asOfUtc || undefined}></az-text>
```

Comment:
We discussed this - you set the value in .show(), not here

thats why it does not work